src/java/test/org/apache/zookeeper/test/AsyncHammerTest.java

```diff
@@ -61,6 +61,7 @@ protected void restart() throws Exception {
         qb.startServers();
     }
 
+    @SuppressWarnings("JUnit4TearDownNotRun")
```

Comment:
Why not annotate with After instead?
Does it break any test?

Reply:
Because it's called from test bodies directly. I've fixed that by making the methods private instead of adding suppression.

Reply:
Wow, that's crazy. We must fix this sometime.
Do you think it could fit into this PR?

Reply:
Fixed what?

Reply:
Annotate with After and remove the call the tests.

Reply:
I would keep it as is for symmetry with setup() method, that couldn't be declared `@Before` because it has a parameter.